setup.sh

```diff
@@ -0,0 +1,15 @@
+ufw allow 32001/tcp
```

Comment:
This is nice.  You might want to add a reference to it in `setup.md`, too.

Reply:
I though I did.